keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt

```diff
@@ -0,0 +1,70 @@
+package com.netflix.spinnaker.keel.echo
+
+import com.netflix.spinnaker.keel.api.AccountAwareLocations
+import com.netflix.spinnaker.keel.api.Locatable
+import com.netflix.spinnaker.keel.api.Monikered
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.events.ResourceHealthEvent
+import com.netflix.spinnaker.keel.events.ResourceNotificationEvent
+import com.netflix.spinnaker.keel.notifications.ClusterViewParams
+import com.netflix.spinnaker.keel.notifications.NotificationMessage
+import com.netflix.spinnaker.keel.notifications.NotificationScope
+import com.netflix.spinnaker.keel.notifications.NotificationScope.RESOURCE
+import com.netflix.spinnaker.keel.notifications.Notifier
+import com.netflix.spinnaker.keel.notifications.Notifier.UNHEALTHY
+import com.netflix.spinnaker.keel.veto.unhealthy.UnsupportedResourceTypeException
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.ApplicationEventPublisher
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+
+
```

Comment:
This class creates the nicely formatted message to send.

@erikmunson I had to remove the duration unhealthy. We can bring it back, but I want to think more deeply about how to save that because we won't be recording it anymore. It would just be recorded for the notification at this point